contracts/apps/disputable/IAgreement.sol

```diff
@@ -5,14 +5,10 @@
 pragma solidity ^0.4.24;
 
 import "../../lib/token/ERC20.sol";
-import "../../lib/arbitration/IArbitrator.sol";
-import "../../lib/arbitration/IAragonAppFeesCashier.sol";
 
 
 contract IAgreement {
 
-    event Signed(address indexed signer, uint256 settingId);
-    event SettingChanged(uint256 settingId);
     event DisputableAppActivated(address indexed disputable);
     event DisputableAppDeactivated(address indexed disputable);
     event CollateralRequirementChanged(address indexed disputable, uint256 collateralRequirementId);
```

Comment:
Perhaps we can also remove the `DisputableAppActivated`, `DisputableAppDeactivated`, and `CollateralRequirementChanged` events (and associated functions)?

For me these now seem like "implementation details" that Disputables don't need to worry about.